docs/API/v3/search-query-service-resource.md

```diff
@@ -0,0 +1,151 @@
+
+If `prerelease` is not provided, pre-release packages are excluded.
+
+The `semVerLevel` query parameter is used to opt-in to SemVer 2.0.0 packages. If this query parameter is excluded, only
+SemVer 1.0.0 compatible packages will be returned. If `semVerLevel=2.0.0` is provided, both SemVer 1.0.0 and SemVer 2.0.0
+compatible packages will be returned. See the [SemVer 2.0.0 protocol](https://github.com/NuGet/Home/wiki/Semver-2.0.0-Protocol)
+for more information.
+
+### Response
+
+The response is JSON document containing up to `take` search results. Search results are grouped by package ID.
+
+Search results returned by nuget.org looks something like this, when using `q=NuGet.Versioning&prerelease=false`:
+
+[!code-JSON [search-result.json](./_data/search-result.json)]
+
+The root JSON object has the following properties:
+
+Name      | Type             | Required | Notes
+--------- | ---------------- | -------- | -----
+totalHits | integer          | yes      | The total number of matches, disregarding `skip` and `take`
+data      | array of objects | yes      | The search results matched by the request
+
+### Search result
+
+Each item in the `data` array is a JSON object comprised of a group of package versions sharing the same package ID.
+The object has the following properties:
+
+Name           | Type                       | Required | Notes
+-------------- | -------------------------- | -------- | -----
+id             | string                     | yes      | The ID of the matched package
+version        | string                     | yes      | The full SemVer 2.0.0 version string of the package (could contain build metadata)
+description    | string                     | no       | 
+versions       | array of objects           | yes      | All of the versions of the package matching the `prerelease` parameter
+authors        | string or array of strings | no       | 
+iconUrl        | string                     | no       | 
+licenseUrl     | string                     | no       | 
+owners         | string or array of strings | no       | 
+projectUrl     | string                     | no       | 
+registration   | string                     | no       | The absolute URL to the associated [registration index](registration-base-url-resource.md#registration-index)
+summary        | string                     | no       | 
+tags           | string or array of strings | no       | 
+title          | string                     | no       | 
+totalDownloads | integer                    | no       | This value can be inferred by the sum of downloads in the `versions` array
+verified       | boolean                    | no       | A JSON boolean indicating whether the package is [verified](../../reference/id-prefix-reservation.md)
+
+On nuget.org, a verified package is one which has a package ID matching a reserved ID prefix and that package owned by
```

Comment:
perhaps drop `that package` from the sentence > `and owned by`

Reply:
Fixed.